genvm/templates/multisig/types.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // SpawnArguments contains a collection with PublicKeys.
 type SpawnArguments struct {
-	PublicKeys []core.PublicKey
+	PublicKeys []core.PublicKey `scale:"max=10"` // update StorageLimit if it changes.
```

Comment:
does the same restriction need to be in `MultiSig` as well?

Reply:
this is what is being decoded from user, that stuff is internal state

Reply:
actually we may enforce it in scalegen, then every collection will have to specify this